### PR TITLE
Fixes #2322

### LINKED ISSUE #2322: Delete order notes (and related meta) when wiping all the data on uninstall
### Reproduction Steps

+ create an order
+ add, e.g., `define( 'LLMS_REMOVE_ALL_DATA', true );` to your wp-config.php 
+ uninstall and delete LifterLMS
+ check the `comments` and `commentmeta` table in the db.

### Expected Behavior

+ I'd expect to see no `llms_order_note` comment types and meta relative to those notes.

### Actual Behavior

+ `llms_order_note` comment types and meta relative to those notes are still there

### Error Messages / Logs

n/a

### System and Environment Information

LifterLMS 7.0.1

### This issue has be recreated:
+ [x] Locally
+ [ ] On a staging site
+ [ ] On a production website
+ [x] With only LifterLMS and a default theme

### Browser, Device, and Operating System Information

n/a

### PR DESCRIPTION
**Pull request by actuallyakash**
## Description
Added query to remove order notes on Plugin Uninstallation.

Fixes #2322

## How has this been tested?
Manually, on local machine.

## Types of changes
Bug fix

## Checklist:
- [x] My code has been tested.
- [x] My code passes all existing automated tests. 
- [x] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

**Review by eri-trabiccolo — CHANGES_REQUESTED** [review]
see https://github.com/gocodebox/lifterlms/issues/2322#issuecomment-1484868450
Also, can you update the file header version's placeholder?
from 
`@version 6.0.0 `
to
`@version [version]`

**Comment by actuallyakash** [comment]
This is done!

File: .changelogs/fix_2322.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+significance: patch
+type: added
+comment: WPDB query to remove order notes from comments table
+links:
+  - "#2322"
+entry: Added query to remove order comments.
```
Comment (eri-trabiccolo) [inline .changelogs/fix_2322.yml]:
```suggestion
entry: Added query to remove order comments on plugin uninstall when the constant `LLMS_REMOVE_ALL_DATA` is set to `true`.
```

**Review by eri-trabiccolo — CHANGES_REQUESTED** [review]

File: .changelogs/fix_2322.yml (hunk 1 above)
Comment (actuallyakash) [reply .changelogs/fix_2322.yml]: thanks :D

Done.

**Review by eri-trabiccolo — APPROVED** [review]

**Pull request merged**